### ISSUE TITLE
Installed, how to configure?

### ISSUE THREAD

**Issue by FrothyDog** [stabbylambda/homebridge-quietcool#2]
I just installed a quiet cool fan, and have the plugin installed and running, but see no documentation for how to configure this. Is there a sample config.json you can share?

**Comment by FrothyDog** [comment]
I have attempted to simply drop the package.json file in my homebridge directory, and it seems to run, but nothing is appearing in Home.

**Comment by stabbylambda** [comment]
The config for this plugin just requires the IP address of the main controller:

```
    "platforms": [{
        "platform": "QuietCool",
        "name": "QuietCool",
        "ip": "10.0.0.151"
    }]
```

I should probably update the README so it contains...something. 😄

**Comment by FrothyDog** [comment]
Yep, that did the trick. Got the fan working and I'm loving this plugin. Thanks!

**Comment by alexmalek** [comment]
I was able to get it working - somewhat through brute force finding the IP. Is there an easy way to get the IP of the device? Nice work building the plugin btw
